core/system/src/unvic.c

```diff
@@ -94,7 +94,7 @@ static void unvic_acl_check(int irqn)
 void unvic_isr_set(uint32_t irqn, uint32_t vector, uint32_t flag)
 {
     /* verify IRQ access privileges */
-    unvic_acl_check(irqn);
+    unvic_acl_check(irqn, g_active_box);
```

Comment:
What's the reason for adding a g_active_box parameter versus directly accessing g_active_box in unvic_acl_check?